libs/utils/utils.js

```diff
@@ -87,6 +87,9 @@ const MILO_BLOCKS = [
   'share',
   'reading-time',
 ];
+
+const MILO_BLOCKS_PRELOADS = { marquee: ['/utils/decorate.js'] };
```

Comment:
Do we want to maintain a manual list of preloads for blocks? While this works as of right now, dependencies change, get added - removed - refactored and we might hurt performance eventually if we start preloading assets that might never get used.

I'm not convinced this is a great long term solution, but a quick band-aid that will lead to issues down the road.

Reply:
Yes, this is a temporary fix. Implementing a comprehensive solution will require a larger effort, potentially an epic. This approach aligns with the performance tiger team's process. Try out various solutions to identify what effectively improves performance.

CC: @jedjedjedM, @auniverseaway